packages/subiquity_client/test/subiquity_client_test.dart

```diff
@@ -120,16 +120,15 @@ void main() {
         password: '',
       );
 
-      // TODO: Re-enable once we figure out why _client.send() sometimes hangs in setGuidedStorage()
-      // try {
-      //   await _client.setGuidedStorage(gc); // should throw
-      //   // ignore: avoid_catches_without_on_clauses
-      // } catch (e) {
-      //   expect(
-      //       e,
-      //       startsWith(
-      //           'setGuidedStorage({"disk_id":"invalid","use_lvm":false,"password":""}) returned error 500'));
-      // }
+      try {
+        await _client.setGuidedStorage(gc); // should throw
```

Comment:
Instead of a try…catch block, shouldn't we use `expect` and `throwsA` with a `predicate()`?